src/components/Availability.js

```diff
@@ -3,6 +3,16 @@ import HelpDialog from "./HelpDialog";
 export default function Availability({ entry }) {
     if (!entry.hasAppointments) {
         return <div>No availability.</div>;
+    } else if (
+        entry.totalAvailability &&
```

Comment:
That's a good a good question. I don't see that data, but that change came from here https://github.com/livgust/macovidvaccines.com/commit/e777424d262689f77ae4d35720d606d43995b271
I pulled it in to avoid a merge conflicts.

Reply:
LOL of course it's something I wrote 🤦 thanks!